collector/scripts/twitter.py

```diff
@@ -0,0 +1,95 @@
+"""
+Tweepy implementation of twitter reader. Requires the 4 twitter keys to operate.
+"""
+
+import tweepy
+import os, time
+import pandas as pd
+import json
+from .utils import tokenize, ada_v2_cost
+
+def twitter():
+    #get user and number of tweets to read
+    username = input("user timeline to read from (blank to ignore): ")
+    searchQuery = input("Search term, or leave blank to get user tweets (blank to ignore): ")
+    tweetCount = input("Gather the last number of tweets: ")
+    
+    #read your API kes to call the api.
```

Comment:
```suggestion
    #read your API keys to call the api.
```

We should check the existence of the keys before invocation and throw an error if any are missing - right now we just try to do the OAuth and hope for the best - if anything the `auth` and `api` should be inside of the try-catch block as well so at least that gets handled.

Reply:
Good point.